src/components/PeopleList/PeopleList.tsx

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+/* eslint-disable jsx-a11y/click-events-have-key-events */
+import React from 'react';
+import cn from 'classnames';
+import { Person } from '../../types/Person';
+
+type Props = {
+  people: Person[],
+  onClick: (person: Person) => void,
```

Comment:
I think it's better to name `onClick` as `onSelected`

Reply:
Yes